src/editor/CSSInlineEditor.js

```diff
@@ -151,6 +151,27 @@ define(function (require, exports, module) {
         return html;
     }
 
+    function checkProvider(hostEditor){
+
```

Comment:
I see that you're duplicating a bunch of code with `htmlToCSSProvider`.  Let's refactor the common bits out of the two functions into a third function, so we can remove the duplication.